azext_iot/iothub/providers/message_endpoint.py

```diff
@@ -42,7 +42,12 @@ def __init__(
         rg: Optional[str] = None,
     ):
         super(MessageEndpoint, self).__init__(cmd, hub_name, rg, dataplane=False)
-        self.support_cosmos = hasattr(self.hub_resource.properties.routing.endpoints, "cosmos_db_sql_collections")
+        # Temporary flag to check for which cosmos property to look for.
+        self.support_cosmos = 0
```

Comment:
nit: Just for readability's sake - could we use an enum (either scoped to this or another shared module) rather than `0`, `1`, `2`, etc?